cli/cmd/install-cni-plugin.go

```diff
@@ -184,6 +191,48 @@ func (options *cniPluginOptions) buildValues() (*cnicharts.Values, error) {
 		return nil, err
 	}
 
+	if val, ok := options.valuesOverrides["enablePSP"]; ok {
+		if err := parseYAMLValue(val, &installValues.EnablePSP); err != nil {
+			return nil, err
+		}
+	}
```

Comment:
Any reason for picking just a few of the options that we have available? For example, doing `--set useWait` won't work here. The solution feels a bit incomplete because of this, and it would make for a wonky user experience when you have no idea which options are supported with `--set`.

Reply:
https://github.com/linkerd/linkerd2/blob/775dc9fbd9d9e1eb043f0ca613a66cdba8e316de/cli/cmd/install.go#L365-L380

Wonder if we can take a similar approach as we do in `install` and simply coalesce the values.

Reply:
I am also thinking to do it this way because the way I have done, it look messy.